css-positioning/absolute.css

```diff
@@ -0,0 +1,37 @@
+.meme {
+  width: 300px;
+  height: 300px;
+  background-image: url('images/one-does-not-simply.jpg');
+  background-size: cover;
+  display: flex;
```

Comment:
This element should be set to position relative so that it constrains any absolutely positioned children